src/components/changeset/map_options.js

```diff
@@ -2,234 +2,151 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
-import { importChangesetMap } from '../../utils/cmap';
 import { updateStyle } from '../../store/map_controls_actions';
-import { Dropdown } from '../dropdown';
+
+// helper functions for adding/removing elements from an array when a
+// checkbox is toggled
+const add = (arr, elem) => {
+  let set = new Set(arr);
+  set.add(elem);
+  return [...set];
+};
+
+const remove = (arr, elem) => {
+  let set = new Set(arr);
+  set.delete(elem);
+  return [...set];
+};
+
+const toggle = (arr, elem) => {
+  return arr.indexOf(elem) === -1 ? add(arr, elem) : remove(arr, elem);
+};
 
 class MapOptions extends React.PureComponent {
-  state = {
-    actions: true,
-    type: true,
-    mapStyle: true,
-    user: true
-  };
   layerOptions = [
-    {
-      label: 'Mapbox Satellite',
```

Comment:
> The Bing aerial imagery layer is just raster imagery; whereas Mapbox Satellite included a vector overlay containing boundaries, city and town names, etc which was helpful for getting your bearings when viewing a changeset. In the future I'm hoping to add an optional overlay similar to this to the Bing imagery layer.

If you end up creating something like this, iD might be able to use it as a replacement for the existing [Location Overlay](https://github.com/openstreetmap/iD/blob/23ef2e5e518f56a443e562bad94bea5ea6a816d2/data/manual_imagery.json#L27), which is a custom Mapbox tileset that needs to be updated manually once in a while.

Reply:
Good to know, I will keep that in mind. For OSMCha this would probably be a vector tileset and accompanying stylesheet that could be overlaid on top of the raster basemap. It looks like iD's locator overlay is a raster tileset of PNGs with alpha channels. Do you know if iD can render vector tiles? I know it uses a custom SVG-based rendering engine (not MapLibre or Leaflet), just wondering if it already supports parsing and using MVTs.